apps/api/src/controllers/pushNotifications.ts

```diff
@@ -0,0 +1,40 @@
+import webpush, { PushSubscription } from 'web-push';
+import { env } from "../configs/env.config";
+import { scheduleJob } from 'node-schedule';
+
+// Configure web-push
+webpush.setVapidDetails(
+  'mailto:mohit@teziapp.com',
+  env.VAPID_PUBLIC_KEY,
+  env.VAPID_PRIVATE_KEY
+);
+
+// In-memory storage for subscriptions (replace with database in production)
+const pushSubscriptions = new Set<PushSubscription>();
+
+export function addSubscription(subscription: PushSubscription) {
+  pushSubscriptions.add(subscription);
+}
+
+export function removeSubscription(subscription: PushSubscription) {
+  pushSubscriptions.delete(subscription);
+}
+
```

Comment:
If multiple instances of backend are running in production, this will lead to same notification being sent to user multiple times.